assets/js/index.js

```diff
@@ -0,0 +1,97 @@
+let library = setupLibrary();
+
+if (!localStorage.uniqueId) {
+  localStorage.uniqueId = 1; 
+}
+
+function setupLibrary() {
+  let library;
+  if(localStorage.library) {
+    library = JSON.parse(localStorage.library).map((e) => Object.assign(new Book(), e));
+  } else {
+    library = [];
+  }
+  return library;
+}
+
+function addBookToLibrary() {
+  const form   = document.getElementById("book-form");
```

Comment:
This variable `form` wasn't used anywhere (per my linter and testing the app with it commented). We can perhaps safely delete this.